src/main/java/ch/njol/skript/expressions/ExprLocationFromVector.java

```diff
@@ -46,11 +48,10 @@
 public class ExprLocationFromVector extends SimpleExpression<Location> {
 
 	static {
-		// TODO fix slowdowns and enable again, for now nuked for greater good
-//		Skript.registerExpression(ExprLocationFromVector.class, Location.class, ExpressionType.SIMPLE,
-//				"%vector% [to location] [in] %world%", "location (from|of) %vector% [(from|in)] %world%",
-//				"%vector% [to location] [in] %world% with yaw %number% and pitch %number%",
-//				"location (from|of) %vector% [(in|from)] %world% with yaw %number% and pitch %number%");
+		Skript.registerExpression(ExprLocationFromVector.class, Location.class, ExpressionType.SIMPLE,
+				"%vector% [to location] in %world%", "location [(from|of)] %vector% (in|from) %world%",
+				"%vector% [to location] in %world% with yaw %number% and pitch %number%",
+				"location [(from|of)] %vector% (in|from) %world% with yaw %number% and pitch %number%");
```

Comment:
```suggestion
				"%vector% [to location] in %world%", "location (from|of) %vector% in %world%",
				"%vector% [to location] in %world% with yaw %number% and pitch %number%",
				"location (from|of) %vector% in %world% with yaw %number% and pitch %number%");
```
I don't think `location {_x} in world` is a great syntax, I think the from/of should be required.
I also don't like `from world`, I think only having `in` is more natural.

Reply:
I didn't wanna change the original syntax if I could help it, but sure if you think that's better.